src/main/java/com/github/maleksandrowicz93/cqrsdemo/student/StudentConfig.java

```diff
@@ -0,0 +1,17 @@
+package com.github.maleksandrowicz93.cqrsdemo.student;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+class StudentConfig {
+
+    @Bean
+    StudentFacade studentFacade(StudentRepository studentRepository) {
+        return new StudentFacade(new StudentQueryHandler(studentRepository),
```

Comment:
jakby handlery były beanami, to w przyszłości łatwiej można byłoby wpiąć w nie inne mechanizmy, np. aspekty